gems/sorbet-runtime/lib/types/types/union.rb

```diff
@@ -42,6 +42,11 @@ def name
       end
     end
 
+    # @override Base
+    def recursively_valid?(obj)
```

Comment:
Seems like the non-enumerable variants of `recursively_valid?` don't have any test coverage - am I missing something?

Reply:
You're right—I've gone ahead and updated the `types` test now to test not only both code paths but ensure that the returned error messages are identical in every test where we'd expect them to be the same.